### DISCUSSION TITLE
Can't send prompt through public address from local network

### DISCUSSION THREAD

**Discussion by myxgenius** [danny-avila/LibreChat#1471] [Troubleshooting] [answered]
Hello, 

I used docker compose up to start the web server, and set port forwarding in my home's AT&T fiber router.

- I can access the web UI normally through my home router's **public ip** address (e.g. 99.12.34.56) when I'm **outside of my home network** such as mobile network or school Wifi.
- I can access the web UI normally **inside my home network** if I'm using my computer's **wifi ip address** (e.g. 10.0.0.123) within the router network, or locally (127.0.0.1)
- But when I try to access the web UI **inside my home network** but using my router's **public ip address** (e.g. 99.12.34.56), the web UI would load and login normally, but as soon as I click the send button for a prompt, the UI hangs (like it's waiting for the answer). I then disconnected from home wifi and refresh the site again, it doesn't look like it received my last prompt.

Anyone knows what prevents the prompt being sent in this case?

Also, Happy new year!

**Comment by peterhoang** [comment, marked as answer]
You could look into using ngrok. It's free for one endpoint. Althought, there's a weird bug: with Dall-e, it generates the image twice per prompt. Not sure if it's my ngrok config or it's a bug with LibreChat.

**Reply by fuegovic** [reply]
That is a known bug, to clarify, the image is not generated twice but only displayed twice, so the cost isn't affected. A fix will come for that.
